CONTRIBUTORS

```diff
@@ -6,7 +6,8 @@
   Fernando Gonzalez-Herrera (translation of linear algrebra)
   Fernando Diaz (review of linear algebra)
   Gustavo Velasco-Hernández (review of linear algebra)
-  
+  Juan P. Chavat (translation of Supervised Learning)
+
```

Comment:
Fernando Gonzalez-Herrera (review of Supervised Learning)
  Fernando Diaz (review of Supervised Learning)

Reply:
Also, add @alonmar as reviewer.

Reply:
Alonso Melgar-Lopez (Review of Supervised Learning)

Reply:
Sorry for forgetting it. Done.